src/docs/OrchardCore.Commerce.Docs.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could you explain why we need the csproj file?
Nothing in it will be built, but it will create a useless NuGet package once  #151 is merged.

Reply:
> Could you explain why we need the csproj file? 
To make it appear in the solution.

> Nothing in it will be built, but it will create a useless NuGet package once #151 is merged.
It shouldn't be built because of the `<IsPackable>false</IsPackable>`

Reply:
Okay then.

Reply:
Unfortunately I found out after merge that this breaks `dotnet pack`, so I have to delete the csproj file after all.
see https://github.com/OrchardCMS/OrchardCore.Commerce/runs/7962504898?check_suite_focus=true#step:4:4202